h/views/account_signup.py

```diff
@@ -29,31 +25,21 @@ def __init__(self, request):
     )
     def get(self):
         """Render the empty registration form."""
-        self._redirect_if_logged_in()
-
-        return {
-            "js_config": self._js_config(),
-        }
-
-    def _js_config(self) -> dict[str, Any]:
-        csrf_token = get_csrf_token(self.request)
```

Comment:
Moved the `_js_config()` helper down to the bottom of the class where we usually put helpers.